pyproject.toml

```diff
@@ -49,7 +50,7 @@ all = ["cloudpathlib[azure]", "cloudpathlib[gs]", "cloudpathlib[s3]"]
 
 [tool.black]
 line-length = 99
-target-version = ['py37', 'py38', 'py39', 'py310', 'py311']
+target-version = ['py37', 'py38', 'py39', 'py310', 'py311', 'py312']
```

Comment:
My understanding is that in general, we don't need this config anymore because the default does something reasonable (infer from the package's Python version specification).

https://github.com/psf/black/pull/3219